akka-docs/src/main/paradox/distributed-data.md

```diff
@@ -22,24 +22,7 @@ to see what this looks like in practice.
 
 ## Introduction
 
```

Comment:
above there is a link:
```
@ref:[distributed-data](typed/distributed-data.md)
```
that should be 
```
@ref:[Distributed Data](typed/distributed-data.md)
```

but maybe that is already changed elsewhere?

Reply:
yes, though I learned if you use the label name, paradox adds the text for you. But I like the explicit better :)